ui/app/components/sidebar/user-menu.hbs

```diff
@@ -39,12 +39,13 @@
                   </LinkTo>
                 </li>
               {{/if}}
-              <li class="action">
+              <li class="action" id="container">
```

Comment:
FWIW I'd probably be more specific in the future. Something like `copy-token-action` - we don't want to accidentally introduce any naming collisions (ids have to be unique, so I think linter catches that, but just my two cents!)